### DISCUSSION TITLE
32 button split helper

### DISCUSSION THREAD

**Discussion by mkaito** [WhiteMagic/JoystickGremlin#511] [Q&A] [answered]
Hi!

I have two daisy chained Virpil devices, each with around 80 buttons total. I was looking to get into Elite: Dangerous, and found it can only handle 32 per device. So I need to split them up across multiple vJoy devices. Is there some way to have JG "just do it" for me, or do I need to manually assign each button?

**Comment by WhiteMagic** [comment]
There is only the option that does it for the fist vJoy device with the "1:1 Mapping" menu item. However, for additional vJoy devices you will have to do the remaps yourself. Though the remap action should pick the next "unused" input when added to speed up the process a bit.

**Reply by mkaito** [reply]
That would work if all my vJoy devices had only 32 buttons. But I've set them to 128 because I mainly use... less ancient games.

Could I manually edit the mapping file to speed up the process a bit?

**Reply by WhiteMagic** [reply, marked as answer]
Yeah, that's definitely doable. The profile is just an XML with a format that should be quite self-explanatory. You can either copy-paste entries and adjust the vjoy and input numbers. The fields for all the physical inputs should already exist. You could also write some code to generate such a profile though that might be more work than a bit of copy&paste.

**Reply by mkaito** [reply]
Sounds like a plan.

Out of curiosity, is this something you'd consider as a feature request? Seems like it might be useful.

**Reply by WhiteMagic** [reply]
Yes that's kind of what #238 is about and something I want to look at for R14 at some stage.
